lib/dai-plugin-mcd/src/CdpType.js

```diff
@@ -0,0 +1,33 @@
+
+
+export default class CdpType {
```

Comment:
CollateralType is a less ambiguous name than CdpType, i think

Reply:
My thinking was that since its possible for two different Ilks to have the same underlying collateral, CollateralType could be a little confusing.  I don't love Cdp type, but that seems to be the colloquial name for an Ilk.  How do you feel about changing the name of CollateralService to CdpTypeService?

Reply:
And also the `getAnnualSavingsRate` and `getSystemWideDebtCeiling` functions could be moved to their own service called something like the `systemDataService`, since they're not related to a specific Ilk

Reply:
> How do you feel about changing the name of CollateralService to CdpTypeService?
I don't love this either, but sure

> And also the getAnnualSavingsRate and getSystemWideDebtCeiling functions could be moved to their own service called something like the systemDataService, since they're not related to a specific Ilk

Sounds good to me